samples/Java/azure-servicebus/TopicClientQuickstart/src/main/java/com/microsoft/azure/servicebus/samples/topicclientquickstart/TopicClientQuickstart.java

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.servicebus.samples.topicclientquickstart;
+
+import com.microsoft.azure.servicebus.*;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.apache.commons.cli.*;
+import org.apache.log4j.*;
+
+public class TopicClientQuickstart {
+    // Connection String for the namespace can be obtained from the Azure portal under the
+    // 'Shared Access policies' section.
+    private static String connectionString = System.getenv("SB_SAMPLES_CONNECTIONSTRING");
+    private static String topicName = System.getenv("SB_SAMPLES_TOPICNAME");
+    private static String subscriptionName = System.getenv("SB_SAMPLES_SUBSCRIPTIONNAME");
+
+    private static ITopicClient topicClient;
+    private static ISubscriptionClient subscriptionClient;
+    private static int totalSend = 100;
```

Comment:
>100 [](start = 35, length = 3)

Maybe change to 10? Too many logger messages (send/receive) on the screen might intimidate the first time user.

Reply:
ok